ts/Dashboards/Dashboard.ts

```diff
@@ -26,6 +26,7 @@
 
 import type JSON from '../Core/JSON';
 
+import DashboardAccessibility from './Accessibility/DashboardsAccessibility.js';
```

Comment:
Should the `DashboardsAccessibility` be a part of the `dashboards`  module or should it be a separate module to include if needed?

Reply:
Oystein suggested to keep it In the `Dashboard/Accessibility` directory, so I named it to be intuitive. I'm open for any suggestions.